src/sentry/search/snuba/executors.py

```diff
@@ -93,6 +93,11 @@ class Clauses(Enum):
     WHERE = auto()
 
 
+# we cannot use snuba for these fields because they require a join with tables that don't exist there
+# if we ever see these fields, we will use postgres to get the group_ids before sending back to ClickHouse
+POSTGRES_ONLY_SEARCH_FIELDS = ["bookmarked_by", "linked", "subscribed_by"]
```

Comment:
Seems ok, these tables are usually small enough that this will be fine. I could imagine that possibly `linked` might end up with a lot of rows due to an automation, although in the past when I checked it was still pretty small